### ISSUE TITLE
New `crates.io` release

### ISSUE THREAD

**Issue by nokome** [r-lib/tree-sitter-r#28]
It would be great if a new release could be made on [crates.io](https://crates.io/crates/tree-sitter-r). In particular, one that includes the expanded semver range in `Cargo.toml`:

https://github.com/r-lib/tree-sitter-r/blob/c19e54de252d5573cc2a762a030957074526fe99/Cargo.toml#L26

Without this change I am still having to use this repo as a Cargo `git` dependency, 

```toml
tree-sitter-r = { git = 'https://github.com/r-lib/tree-sitter-r', rev = "016051ef9ec6f31a42043767316eaae98f8e8f2c" }
```

which is fine except that for us it is breaking our Windows CI and avoiding Cargo `git` dependencies looks like the easiest way to avoid this failure.

**Comment by jimhester** [comment]
Sure, published 0.19.5 now, let me know if you need anything else!

**Comment by nokome** [comment]
Thanks for your quick attention to this, much appreciated 🙏.
